### PR TITLE
docker-machine-driver-qemu 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by JounQin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by JounQin** [comment]
ping @SMillerDev

**Comment by chenrui333** [comment]
- the upstream project is not well maintained in terms of releases and etc, see https://github.com/machine-drivers/docker-machine-driver-qemu/issues/8#issuecomment-419704756

**Comment by JounQin** [comment]
But it still works? Of course, if you don't want to include it into core I can extract it to my own tap.

File: Formula/d/docker-machine-driver-qemu.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class DockerMachineDriverQemu < Formula
+  desc "Docker machine driver for pure qemu/kvm - no libvirt"
+  homepage "https://github.com/machine-drivers/docker-machine-driver-qemu"
+  url "https://github.com/machine-drivers/docker-machine-driver-qemu/archive/0324171328f7bdf2f1e7803dbdc3b18db457c786.tar.gz"
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/d/docker-machine-driver-qemu.rb:4]: that is the issue, we dont ship un-tagged commits in homebrew-core

**Comment by chenrui333** [comment]
> But it still works? It's just stable enough to me. Of course, if you don't want to include it into core I can extract it to my own tap.

you are free to host in your own tap. I am gonna close this PR for now. (upstream does not maintain the regular releases, which is an issue for core)

**Pull request closed without merging**